openwisp_users/mixins.py

```diff
@@ -35,7 +35,7 @@ class OrgMixin(ValidateOrgMixin, models.Model):
     - implements ``_validate_org_relation`` method
     """
     organization = models.ForeignKey('openwisp_users.Organization',
-                                     verbose_name=_('organization'))
+                                     verbose_name=_('organization'), on_delete=models.CASCADE)
```

Comment:
please move the `on_delete` parameter to the subsequent line for consistency with the previous lines